src/components/admin/form-components/ListSelector.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+export default class ListSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange(event, ...restArgs) {
+    this.props.update(restArgs[1]);
```

Comment:
If you're only using the second argument, and it also seems mandatory, just define the second argument instead of using the `rest` operator

Reply:
Yeah, this was a braindead copy paste - TODO